cqrs-example/cqrs-example.cabal

```diff
@@ -24,12 +24,12 @@ Executable cqrs-example
   Main-is:           CQRSExample/Main.hs
   Ghc-options:       -Wall
   Build-depends:     base >= 4.8 && < 5
-                   , aeson >= 0.8 && < 0.9
+                   , aeson >= 0.8
```

Comment:
Ok, I don't think those will be any problem. I'll see if I can't do a commit on master with those later on today.